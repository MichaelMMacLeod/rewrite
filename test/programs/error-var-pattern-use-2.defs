(def (twice $x $x) constructor)