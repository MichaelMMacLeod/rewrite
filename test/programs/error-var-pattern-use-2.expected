error[E006]: variable occurs more than once in pattern
./test/programs/error-var-pattern-use-2.defs:1:13
   |
 1 | (def (twice $x $x) constructor)
   |             ^^ use #1
./test/programs/error-var-pattern-use-2.defs:1:16
   |
 1 | (def (twice $x $x) constructor)
   |                ^^ use #2
help: a variable may occur at most once in a definition's pattern
